grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -193,7 +193,7 @@ class CycleCountService {
                         "Lot Number": item.inventoryItem.lotNumber,
                         "Expiration Date": item.inventoryItem.expirationDate
                                 ? Constants.EXPIRATION_DATE_FORMATTER.format(item.inventoryItem.expirationDate) : "",
-                        "Bin Location": item.binLocation?.locationNumber,
+                        "Bin Location": item.binLocation?.name,
                         "Quantity Counted": "",
                         "Comment": "",
                         "User Counted": "",
```

Comment:
is it intentional that user counted and date counted are always left blank (even if we've set a value for them in the app)? This would be different behaviour than the PDF (not saying that's a bad thing, just observing).